src/scimlfunctions.jl

```diff
@@ -3194,7 +3194,28 @@ function DAEFunction{iip, specialize}(f;
                                  _colorvec, sys)
     end
 end
-
+struct DAEFunction{iip, specialize, F, Ta, Tt, TJ, JVP, VJP, JP, SP, TW, TWt, TPJ, S, S2,
```

Comment:
Duplicated https://github.com/SciML/SciMLBase.jl/blob/4ef1c09f0b7db4ade308d7e819415f2ae84e384a/src/scimlfunctions.jl#L1663 ?

Reply:
yeah. Typo (I copied it so I could see both on the same screen and forgot to delete). I also am working on getting a test for this.